gradle/libs.versions.toml

```diff
@@ -1,5 +1,5 @@
 [versions]
-kotlin-version = "1.9.24"
+kotlin-version = "1.9.23"
```

Comment:
Could you tell me if we can start using 2.0.0?

Reply:
Seems to be a non-trivial change on account of changes to the compiler API.  I'll look into it in a separate task.